src/styles/sidebar.css

```diff
@@ -0,0 +1,73 @@
+.sidebar {
+    position: fixed;
+    top: 0;
+    left: 0;
+    bottom: 0;
+    display: flex;
+    flex-direction: column;
+    width: 300px;
+    height: 100%;
+    background-color: var(--primary-color);
+    color: #fff;
+    z-index: 5;
+    box-shadow: 10px 10px 10px 5px 0.5;
+}
+.sidebar-icon {
+    margin-right:16px;
+    cursor: pointer;
+    user-select: none;
+}
+
+.sidebar-header {
+    display: flex;
+    align-items: center;
+    padding: 16px;
+}
+
+.sidebar-items {
+    display: flex;
+    flex-direction: column;
+    padding: 0 8px;
+}
+
+.sidebar-items .item {
+    display: flex;
+    align-items: center;
+    padding: 16px 8px;
+    border-radius: 5px;
+    transition: var(--info-color) 0.2s;
+    cursor: pointer;
+}
+
+.sidebar-items .item:hover {
+    background-color: var(--info-color);
+}
+
+/* ---- Collasped Sidebar --- */
+.sidebar.collapsed {
+    width: 52px;
+}
+
+.sidebar.collapsed .sidebar-icon {
+    margin-right: 0;
+}
+
+.sidebar.collapsed .sidebar-text,
+.sidebar.collapsed .sidebar-logo {
+    display: none;
+}
+
+.sidebar a {
+    color: #fff;
+}
+
+.sidebar a:link { text-decoration: none; }
+
+
+.sidebar a:visited { text-decoration: none; }
+
+
+.sidebar a:hover { text-decoration: none; }
+
+
+.sidebar a:active { text-decoration: none; }
```

Comment:
Suggestion - you can be more consistent with how you write rules.
Either adopt
```selector { property: value; }```
or 
```selector {
   property: value;
}
```
for single lined rules.

Reply:
Done